compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -62,8 +62,10 @@ class Compiler {
            new ExplicitOuter,       // Add accessors to outer classes from nested ones.
            new ExplicitSelf,        // Make references to non-trivial self types explicit as casts
            new CrossCastAnd,        // Normalize selections involving intersection types.
-           new Splitter),           // Expand selections involving union types into conditionals
-      List(new VCInlineMethods,     // Inlines calls to value class methods
+           new Splitter,           // Expand selections involving union types into conditionals
+           new SpecializeFunction1), // Specialized Function1 by replacing super with specialized super
```

Comment:
I was unsure of what impact the interleaved phases would have here. `DispatchToSpecializedApply` needs to know that the identifier type has a specialized apply before dispatching to it - so the addition to the decls (in `SpecializeFunction1`) needs to happen before this phase's transformation is applied.